data/src/main/java/com/microsoft/azure/kusto/data/KustoOperationResult.java

```diff
@@ -122,7 +128,24 @@ private void createFromV2Response(String response) throws KustoServiceQueryError
             if (jsonNode.isArray()) {
                 jsonArray = (ArrayNode) jsonNode;
                 for (JsonNode node : jsonArray) {
-                    if (node.has(FRAME_TYPE_PROPERTY_NAME) && node.get(FRAME_TYPE_PROPERTY_NAME).asText().equals(DATA_TABLE_FRAME_TYPE_PROPERTY_NAME)) {
+                    if (node.has(FRAME_TYPE_PROPERTY_NAME) && node.get(FRAME_TYPE_PROPERTY_NAME).asText().equals(DATA_SET_COMPLETION_FRAME_TYPE_PROPERTY_VALUE)
```

Comment:
Can do this once
if (node.has(FRAME_TYPE_PROPERTY_NAME) TableType = node.get(FRAME_TYPE_PROPERTY_NAME)